### PR TITLE
[disaster] create new dashboard page

### PR DESCRIPTION
**Pull request by chejennifer**
- Create the new disaster dashboard page
- extract out shared css for shared components
- rename old disaster dashboard related files to disaster_dashboard_v0

### CODE REVIEW

File: server/routes/disasters.py [hunk 1]
```diff
@@ -13,19 +13,52 @@
...
+    if place_dcid in config.metadata.place_dcid:
+      dashboard_config = config
+      break
+  if not dashboard_config:
+    return "Error: no config found"
+
+  place_type = DEFAULT_PLACE_TYPE
+  if place_dcid != DEFAULT_PLACE_DCID:
+    place_type = place_api.get_place_type(place_dcid)
+  place_name = place_api.get_i18n_name([place_dcid]).get(place_dcid, place_dcid)
```
Comment (beets) [inline server/routes/disasters.py:58]: please escape place_dcid before using it as the default returned value.

File: server/tests/subject_page_configs_test.py [hunk 2]
```diff
@@ -52,8 +52,10 @@ def verify_tile(self, tile, stat_vars, msg):
 
   def test_required_fields(self):
     """Tests all configs loaded at server start"""
-    all_configs = []
-    all_configs.append(libutil.get_topic_page_config())
+    all_configs = {}
+    all_configs.update(libutil.get_topic_page_config())
```
Comment (beets) [inline server/tests/subject_page_configs_test.py:56]: nit: missing key for the dict update?

**Review by beets — APPROVED** [review]
changes lgtm

in future, would strongly prefer refactorings to be in separate pr's from new code additions. would have made diffs much easier to track..

File: server/tests/subject_page_configs_test.py (hunk 2 above)
Comment (chejennifer) [reply server/tests/subject_page_configs_test.py:56]: get_topic_page_config() returns a dict

File: server/routes/disasters.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/disasters.py]: good catch, done

File: server/tests/subject_page_configs_test.py (hunk 2 above)
Comment (beets) [reply server/tests/subject_page_configs_test.py:56]: ah, poor function naming on my part then!

**Comment by chejennifer** [comment]
Thanks for the review! Sorry for making the diffs hard to track, will create a separate PR for refactors in the future!

**Pull request merged**